submissions/oyetejuOloyede/DOM/contactForm.js

```diff
@@ -0,0 +1,36 @@
+let submitForm = document.getElementById('submit');
+let fullName = document.getElementById('fullName');
+let email = document.getElementById('email');
+let subject = document.getElementById('subject');
+let textArea = document.getElementById('textArea');
+let fullNameMsg = 'Please enter your Name';
+let emailMsg = "Please enter your Email Address";
+let emailErrorMsg = "Please enter a valid email";
+let subjectMsg = "Please enter a message subject";
+let textAreaMsg = "Please enter your Message";
+
+function valid(regex, field, error1, error2, event) {
+    if(field.value === "") {
+        field.nextElementSibling.innerHTML = error1;
+        event.preventDefault();
+        return false;
+      } else if (!(field.value.match(regex))) {
+          field.nextElementSibling.innerHTML = error2;
+          event.preventDefault();
+          return false;
+          }
+      else {
+        field.nextElementSibling.innerHTML = "";
+        return true;
+      }
+}
+
+function validation(e) {
+    valid(/^[a-z]+$/i, fullName, fullNameMsg, fullNameMsg, e);
```

Comment:
`error1` and `error2` have the same error message. is it suppose to be so?

Reply:
error1 and error2 corrected